JS/Lesson24/saffet/filter.js

```diff
@@ -409,3 +450,11 @@ const usersExercise10 = [
 		email: 'sophia@example.com',
 	},
 ];
+
+const startWithJ = usersExercise10.filter(user => {
+	if(user.name.charAt(0)=== "J" || user.name.charAt(0)=== "j"){
```

Comment:
Use toLowerCase() to make sure that the text is in the same case, then you don't need to write multiple conditions
```suggestion
	if(user.name.charAt(0).toLowerCase() !== "j" ){
```